src/convert.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { convertSelicEntry } from "./convert";
+import { format, compareAsc } from "date-fns";
+
+describe("convert", () => {
```

Comment:
Usually, we use the name of the function/class under test. It might be better to use something like

```suggestion
describe("convertSelicEntry", () => {
```